tests/phpunit/api/v4/Entity/WorkflowMessageTest.php

```diff
@@ -92,21 +98,25 @@ public function getRenderExamples(): array {
   }
 
   /**
+   * Test examples render.
+   *
+   * Only examples tagged phpunit will be checked.
+   *
    * @param string $name
    * @throws \API_Exception
    * @throws \Civi\API\Exception\UnauthorizedException
    * @dataProvider getRenderExamples
    */
-  public function testRenderExamples(string $name) {
-    $example = \Civi\Api4\ExampleData::get(0)
+  public function testRenderExamples(string $name): void {
+    $example = ExampleData::get(FALSE)
       ->addWhere('name', '=', $name)
       ->addSelect('name', 'file', 'data', 'asserts')
       ->execute()
       ->single();
 
-    $this->assertTrue(!empty($example['data']['modelProps']), sprintf('Example (%s) is tagged phpunit. It should have modelProps.', $example['name']));
-    $this->assertTrue(!empty($example['asserts']['default']), sprintf('Example (%s) is tagged phpunit. It should have assertions.', $example['name']));
-    $result = \Civi\Api4\WorkflowMessage::render(0)
+    $this->assertNotEmpty($example['data']['modelProps'], sprintf('Example (%s) is tagged phpunit. It should have modelProps.', $example['name']));
```

Comment:
I think it was the way it was before because if e.g. modelProps doesn't exist, then it doesn't even get to the assert and fails in an uncontrolled way as opposed to a controlled way with the nice message. But almost the same end result.